SparkPost/2web2ui
# SA-2047: Changed expandable behavior

### What Changed

- Changed default state for expandables

### How To Test

- Verify that nav is smaller
- Verify metrics expandable is open
- Verify other expandables are closed
- Verify that the buttons in expandables are smaller

### To Do

- [ ] Update testing
